scripts/datasets/redial/format_redial.py

```diff
@@ -0,0 +1,283 @@
+"""Script to format the ReDial dataset into DialogueKit format.
+
+In addition to formatting the dialogues, this script also process items and
```

Comment:
Also mention that consecutive utterances are merged?

Reply:
Added details on formatting.